recipes-core/images/nilrt-safemode-initramfs.bb

```diff
@@ -22,7 +22,7 @@ IMAGE_INSTALL_NODEPS += "\
 	${NI_PROPRIETARY_SAFEMODE_PACKAGES} \
 "
 
-BAD_RECOMMENDATIONS_append_pn-${PN} += "shared-mime-info"
+BAD_RECOMMENDATIONS:append_pn-${PN} += "shared-mime-info"
```

Comment:
I'm not familiar with `append_pn-${PN}` enough to know if this is correct. Did you check that this the right override placement?

Reply:
I'll double-check after I rebase so this is just the syntax updates.

Reply:
Yep, this was incorrect. It should be `:append:pn-${PN}` based on a quick search on yocto.